assets/js/util/get-context-scroll-top.js

```diff
@@ -0,0 +1,40 @@
+import { ANCHOR_ID_TRAFFIC } from '../googlesitekit/constants';
+
+/**
+ * Gets the y coordinate to scroll to the top of a context element, taking the sticky admin bar, header and navigation height into account.
+ *
+ * @since n.e.x.t
+ *
+ * @param {string} contextID  The ID of the context element to scroll to.
+ * @param {string} breakpoint The current breakpoint.
+ * @return {number} The offset to scroll to.
+ */
+const getContextScrollTop = ( contextID, breakpoint ) => {
+	const contextElement = document.getElementById( contextID );
+	if ( contextID === ANCHOR_ID_TRAFFIC || ! contextElement ) {
```

Comment:
I think we can move this contextID check back to the `DashboardNavigation` component since it's only makes sense there.

```diff
- if ( contextID === ANCHOR_ID_TRAFFIC || ! contextElement ) {
+ if ( ! contextElement ) {
```

Then in the `assets/js/components/DashboardNavigation.js` file:

```diff
 global.scrollTo( {
-	top: getContextScrollTop( hash, breakpoint ),
+	top: hash !== ANCHOR_ID_TRAFFIC ? getContextScrollTop( hash, breakpoint ) : 0,
 	behavior: 'smooth',
 } );
```

Reply:
Perfect!